server/src/main/java/com/senlainc/warsaw/tyurin/util/Constants.java

```diff
@@ -0,0 +1,18 @@
+package com.senlainc.warsaw.tyurin.util;
+
+public class Constants {
+    public static final String PATH_TO_CRAFTSMEN_CSV = "src\\main\\resources\\craftsmen.csv";
+    public static final String PATH_TO_GARAGE_PLACES_CSV = "src\\main\\resources\\garagePlaces.csv";
+    public static final String PATH_TO_ORDERS_CSV = "src\\main\\resources\\orders.csv";
+    public static final String CRAFTSMEN_CSV_HEADER = "id,name,surname";
+    public static final String GARAGE_PLACES_CSV_HEADER = "id,number,space";
+    public static final String ORDERS_CSV_HEADER = "id,price,submissionDate,startDate,completionDate,orderStatus,craftsmenId,garagePlaceId";
+    public static final String PATH_TO_PROPERTIES = "application.properties";
+    public static final String ABILITY_TO_ADD_GARAGE_PLACE = "abilityToAddGaragePlace";
```

Comment:
I did not mention it last time, my bad. Usually properties writen in kebab-case with dot separator for logical split. Also try to avoid "to", "in", "for" etc.
Example: 
`garage-place.add.enabled=true`